storybook/src/stories/person/select-component.stories.mdx

```diff
@@ -12,23 +12,18 @@ import PersonSelect from '@equinor/fusion-react-person/src';
 
 Name                      | Type                        | Default          | Description
 ------------------------- | -------------------------   | --------------   | -----------
-`placeholder`             | `string`                    | `/`              | TextInput placeholder string
-`initialText`             | `string`                    | `/`              | First item in dropdown before any search is done
```

Comment:
`initialText` is removed. I believe it should still be described in the table.

Reply:
initialText is now removed in example instead, its not used in component.